src/app/pages/offline-queue/offline-queue.page.html

```diff
@@ -0,0 +1,46 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button defaultHref="tasks"></ion-back-button>
+    </ion-buttons>
+    <ion-title>
+      Offline queue
+    </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content class="outer-content">
+  <ion-list *ngIf="stagedItems">
+    <ion-item-group
+      *ngFor="let stagedItem of stagedItems; let last = last"
+      [class.last-item]="last"
+    >
+      <ion-item>
+        <ion-label>
+          <h2>
+            Mutation type:
+            <ion-badge color="primary">
+              {{ stagedItem.operationName }}
+            </ion-badge>
+          </h2>
+          <div *ngIf="stagedItem.variables.title">
```

Comment:
Not for now but it could be simple for loop that iterates over variables as the style for all is the same. When we change schema this will silently fail.